addons/mail/data/ir_cron_data.xml

```diff
@@ -42,5 +42,16 @@
             <field name="state">code</field>
         </record>
 
+        <record id="ir_cron_delete_rtc_sessions" model="ir.cron">
```

Comment:
Running every 2 minutes is way too much ... on saas notably this could lead to a lot of resources used for garbage collecting. You should probably lessen its running time, normally running less often should not be an issue.